pkg/util/globalerror/grpc_test.go

```diff
@@ -77,16 +83,24 @@ func TestWrapContextError(t *testing.T) {
 				expectedGrpcCode:   codes.DeadlineExceeded,
 				expectedContextErr: context.DeadlineExceeded,
 			},
+			"grpc.ErrClientConnClosing": {
+				origErr:            status.Error(codes.Canceled, grpcClientConnectionIsClosingErr),
```

Comment:
Could use `grpc.ErrClientConnClosing` here

Reply:
If we use `grpc.ErrClientConnClosing` lint will complain because that error is deprecated.
We can use that error, but we need to ignore lint